README.md

```diff
@@ -66,37 +149,51 @@ The "SessionCheck" module can be loaded in several ways:
     setInterval(function () {
         sessionCheck.triggerSessionCheck();
     }, 60000);
+```
 
 *Details you need to provide:*
 
- - subject - The user currently logged into the RP
+ - invalidSessionHandler - function to be called once any problem with the session is detected, with reason for the invalid sessions and request count included
+ - subject [optional] - Only used with `responseType=id_token` and ForgeRock-only methods. The user currently logged into the RP. If not supplied, subject changes won't trigger the invalidSessionHandler
+ - sessionClaimsHandler [optional] - function to be called after every successful session check, with latest claims and request count included. Only used with `responseType=id_token`.
+ - initialSessionSuccessHandler [optional] - optional function to be called after the first successful session check request.
+ - cooldownPeriod [default: 5] - Minimum time (in seconds) between requests to the opUrl
+
+ForgeRock-specific options:
+ - ssoToken - String representing the user's session within AM. Likely made available as a custom idToken claim
+ - amUrl - The full URL (including path to the base of the realm) of the AM server that issued the ssoToken
+
```

Comment:
Although it has a default value, `ssoTokenName` should probably be mentioned here. But, also see my comments in the `sessionCheck` module definition.